src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Advanced CoreDNS Configuration"
+description = "Here we describe how you can customize the configuration of the managed CoreDNS service in your clusters"
+date = "2018-10-30"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Advanced CoreDNS Configuration
+
+The [CoreDNS addon](https://github.com/coredns/coredns) running inside your cluster has additional configuration options and features that can be customized.
+
+You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `coredns-user-values` and is located in the `kube-system` namespace.
+
+__Note:__ This feature is only available in more recent cluster versions. To check if your cluster version supports customization through the ConfigMap, you can check if the above-mentioned ConfigMap is present.
+
+__Warning:__ Please do not edit any of the other CoreDNS related resources. Only the user ConfigMap is safe to edit.
+
+```nohighlight
+$ kubectl -n kube-system get cm coredns-user-values
+NAME                                   DATA      AGE
+coredns-user-values                    0         11m
+```
+
+On cluster creation the ConfigMap is empty and below-mentioned defaults will be applied to the final CoreDNS deployment. To customize any of the configuration options, you just need to add the respective line(s) in the data field of the user ConfigMap.
+
+## Cache Settings
+
+By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like following.
+
+```yaml
+data:
+  cache: 60
+```
+
+Above setting increases the TTL to 60 seconds.
+
+The cache plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/cache/).
+
+## Additional Proxies
+
+The default proxy entry we set in CoreDNS is
+
+```yaml
+proxy . /etc/resolv.conf
+```
+
+You can add additional proxy entries by adding a proxy list to the user ConfigMap and creating a list item for each proxy entry you want to be added.
+
+```yaml
+data:
+  proxy:
+    - foo.com 1.1.1.1
+    - bar.com 8.8.8.8
```

Comment:
@TheoBrigitte please also check

Reply:
wrong :negative_squared_cross_mark: 

configmap values are always string, so we can have:
* one value
```
  proxy: foo.com 1.1.1.1
```

* one value per line by using multi-line value
```
  proxy: |
    foo.com 1.1.1.1
    bar.com 8.8.8.8
```

Reply:
ah, ok, thanks, wasn't sure